packages/monaco/src/browser/monaco-quick-open-service.ts

```diff
@@ -51,6 +52,9 @@ export class MonacoQuickOpenService extends QuickOpenService {
     @inject(KeybindingRegistry)
     protected readonly keybindingRegistry: KeybindingRegistry;
 
+    @inject(BrowserMenuBarContribution)
+    protected readonly browserMenuBarContribution: BrowserMenuBarContribution;
```

Comment:
wonder whether it works in electron

Reply:
probably we should annotate it as `@optional` and use `BrowserMenuBarContribution | undefined` as a type that it is no-op in electron